COMMUNICATION.md

```diff
@@ -0,0 +1,36 @@
+There are different ways and channels that you can use as a community member or
+maintainer to raise questions, collaborate or to have some fun!
+
+## Slack
+
+The community has a channel on the Packet Slack called
+[#tinkerbell](https://tinkerbell.org/community/slack/). Feel free to join us.
+This is the perfect place to have a first point of contact with the community.
+That's where maintainers are contributors are. It is also the oldest platform
```

Comment:
> That's where maintainers are contributors are.

Needs re-wording right? Maybe "The maintainers and some contributors hang out on it."